var/spack/repos/builtin/packages/synapsetool/package.py

```diff
@@ -32,8 +32,9 @@ class Synapsetool(CMakePackage):
     homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/hpc/synapse-tool"
     url      = "ssh://bbpcode.epfl.ch/hpc/synapse-tool"
 
-    version('develop', git=url, submodules=True)
-    version('0.2.5', git=url, tag='v0.2.5', submodules=True, preferred=True)
+    version('develop', git=url, submodules=True, preferred=True)
+    version('0.3.0', git=url, tag='v0.3.0', submodules=True)
```

Comment:
I think we should make last released `0.3.0` as preferred instead of develop. Because everytime develop will install latest master and hence different version.